### ISSUE TITLE
Prepare JRuby binding for submission to OpenHAB core

### ISSUE THREAD

**Issue by boc-tothefuture** [boc-tothefuture/openhab-jruby#103]
- [ ] Documentation of code
- [ ] Refactor code into multiple classes
- [ ] Create readme.md 
- [ ] The global variables filtering should be in addition to local variables for the GUI use case

**Comment by pacive** [comment]
Do you need any help with this? One thing I think would be beneficial is that the configuration that's currently in the jruby.conf file would be exposed to the UI. In what repo do you have the current code?

**Comment by boc-tothefuture** [comment]
Currently it can't be exposed in the UI.  See this issue for details:  https://github.com/openhab/openhab-core/issues/2034

I started the refactor proces into multiple classes.  A readme on the configuration and and changing the variable filtering to be additive rather modifying the scope would get it pretty close to ready.  

Latest is [here](https://github.com/boc-tothefuture/openhab2-addons/tree/jruby)

**Comment by jimtng** [comment]
Now that the addon has been accepted into openhab, I started playing with what people can do with gui rules.

I wrote this simple test, which worked:

```ruby
java_import org.slf4j.Logger
java_import org.slf4j.LoggerFactory
logger = LoggerFactory.getLogger('org.openhab.model.script.MyRule')

logger.info("Hi Im running from gui #{Time.now}")

$events.send_command('StudyRoom_Light', 'OFF')
```

We might want to include a simple example like this to get people started.

And of course this also works:
```ruby
require 'openhab'
logger.info("Hi Im running from gui #{Time.now}")
StudyRoom_Light.off
```

I believe we should mention this library and show some example code in the main jrubyscripting automation docs. At the very least we should try or ask.

If we want to encourage people to use this form via gui, my question would be, how do we get to the `event` object that's passed to the usual `run` execution block in textual rules?
